doc/nrf/releases_and_maturity/migration/migration_guide_2.7.rst

```diff
@@ -71,6 +71,19 @@ Peripheral samples
   * The CLI command ``fem tx_power_control <tx_power_control>`` replaces ``fem tx_gain <tx_gain>`` .
     This change applies to the sample built with the :ref:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC <CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC>` set to ``n``.
 
+
+Matter
+------
+
+  * With the inheritance of Zephyr's :ref:`zephyr:sysbuild` in the |NCS|, some changes are provided to the Matter samples and applications:
+
+    * :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_BUILD` kconfig option is deprecated and you need to use the :kconfig:option:`SB_CONFIG_MATTER_FACTORY_DATA` Kconfig option instead to enable or disable creating the factory data set during building a Matter sample.
```

Comment:
deprecated is wrong, deprecated seems like it still works, but it does not

Reply:
Because it still works for non-sysbuild (child images) sample variant that as far as I know must be supported. But ok, deprecated may be misleading here.

Reply:
I re-thinked it and "deprecated" is correct here, because in fact it still must be used for the non-sysbuild variants. I will add a one-line comment there.